src/main/scala/org/hombro/maze/Maze.scala

```diff
@@ -0,0 +1,132 @@
+package org.hombro
+
+import scala.collection.mutable
+
+package object maze {
+
+  // maze parameters: we specify number of cells in horizontal and vertical direction
+  // as well as the physical height and width of the maze
+  case class MazeParams(nx: Int, ny: Int)
+
+  // we'll idetify cells by their integer ids
+  type CellID = Int
+  // Set label
+  type Label = Int
+  // we represent side of a cell as an ordered tuple of two integers (c1,c2)
+  // where c1 and c2 are cell ids and c1 < c2
+  type Side = (Int, Int)
```

Comment:
if the Ints are meant to be CellID, then can't you pass the type directly into there?